static/app/components/forms/formField/index.tsx

```diff
@@ -417,12 +417,13 @@ class FormField extends Component<FormFieldProps> {
                     type={saveMessageAlertType}
                     trailingItems={
                       <Fragment>
-                        <Button onClick={this.handleCancelField} size="xs">
+                        <Button onClick={this.handleCancelField} size="xs" type="button">
```

Comment:
lol.. doesn't the button component have the type `button` by default? if not... shouldn't we add it there?

Reply:
I believe there is some reason we decided not to do this.

I think we were worried there may be some buttons that will simply stop working, since they'll no longer submit forms